lib/csl/mapper.rb

```diff
@@ -59,7 +59,7 @@ def csl_doc
         # Add journal information if it exists.
         if pub_hash.key?(:journal)
           if pub_hash[:journal][:name].present?
-            cit_data_hash['type'] = 'article-journal'
+            cit_data_hash['type'] = 'article-journal' unless cit_data_hash['type'] == 'inproceedings'
```

Comment:
Arcadia noticed that we were incorrect setting type for `inproceedings`.

Reply:
this is used when mapping pub_hash to the citation generator right?  so this basically would change for how we format citations going forward

Reply:
I suspect it doesn't actually change any MLA, Chicago, or APA citations, but will only affect Bibtex citations (which are new).